src/components/Schematics/Schematic-Flow.vue

```diff
@@ -1075,9 +1075,7 @@ export default {
               Thanks! We'll get our team on it immediately.
             </div>
             By the way, you can
-            <a
-              target="_blank"
-              href="https://join.slack.com/t/prefect-community/shared_invite/enQtODQ3MTA2MjI4OTgyLTliYjEyYzljNTc2OThlMDE4YmViYzk3NDU4Y2EzMWZiODM0NmU3NjM0NjIyNWY0MGIxOGQzODMxNDMxYWYyOTE"
+            <a target="_blank" href="https://prefect.io/slack"
```

Comment:
any reason we don't want to use `ExternalLink` for this?

Reply:
Hm no reason, just that this predates the external link component and there'd be more styling to do here than I want atm